packages/pycharm-community-bin/pycharm-community-bin.pacscript

```diff
@@ -0,0 +1,39 @@
+name="pycharm-community-bin"
+version="2021.3.3"
+pkgname="pycharm-community"
+gives="pycharm-community"
+maintainer="Emil Aarestrup <emil@aarestrup.nu>"
+description="Python IDE for professional developers aimed at productivity with a keyboard-centric focus."
+url="https://download.jetbrains.com/python/pycharm-community-${version}.tar.gz"
+hash="06ef4152d59b97da1186cdf288588d20d69a689fe75b2140130fa5d8326d07ad"
+
+prepare() {
+true
+}
+
+build() {
+true
+}
+
+install() {
+
+sudo mkdir $STOWDIR/$name
+sudo mv "../$pkgname-$version/" "$STOWDIR/$name"
```

Comment:
Pacstall will link all files from `$STOWDIR/$name` so $pkgname-$version would be at `/$pkgname-$version` if you were to move it at `$STOWDIR/$name/usr/bin/$pkgname-$version` it would be at `/usr/bin/$pkgname-$version`.